.jenkins/.pipeline/lib/build.js

```diff
@@ -1,23 +1,38 @@
 'use strict';
-const {OpenShiftClientX} = require('pipeline-cli')
+const {OpenShiftClientX} = require('@bcgov/pipeline-cli')
 const path = require('path');
 
 module.exports = (settings)=>{
-  const phases = settings.phases
-  const oc=new OpenShiftClientX({'namespace':phases.build.namespace});
+  const phases=settings.phases
+  const options = settings.options
+  const oc=new OpenShiftClientX(Object.assign({'namespace':phases.build.namespace}, options));
   const phase='build'
-  let objects = []
+  var objects = []
+
   const templatesLocalBaseUrl =oc.toFileUrl(path.resolve(__dirname, '../../openshift'))
 
-  objects = objects.concat(oc.processDeploymentTemplate(`${templatesLocalBaseUrl}/jenkins.bc.json`, {
+  objects.push(...oc.processDeploymentTemplate(`${templatesLocalBaseUrl}/build-master.yaml`, {
     'param':{
       'NAME': phases[phase].name,
       'SUFFIX': phases[phase].suffix,
       'VERSION': phases[phase].tag,
       'SOURCE_REPOSITORY_URL': oc.git.http_url,
-      'SOURCE_REPOSITORY_REF': oc.git.ref
+      'SOURCE_REPOSITORY_REF': oc.git.ref,
+      'SOURCE_IMAGE_STREAM_NAMESPACE': phases[phase].namespace,
+      'SOURCE_IMAGE_STREAM_TAG': 'jenkins-basic:prod'
```

Comment:
Should we externalize this value for the image tag?

Reply:
yea that's me being lazy thinking that we'll switch to another cicd tool :P let me update it!